### DISCUSSION TITLE
Update: Major Community Hub updates incoming

### DISCUSSION THREAD

**Discussion by smartcontracts** [ethereum-optimism/community-hub#283] [Announcements]
Hello! And a huge thank you to everyone who has been contributing to this website. We are trying to make this the best resource for information about Optimistic Ethereum and I think we've gone a long way towards that goal.

## Important updates

Core maintainers been discussing an overhaul of a significant portion of the documentation available on the Community Hub. The Hub has, so far, been a collection of articles/pages with no clear target audience. We've attempted to fix this issue in the past by having sections for users and developers, but we've found that the developer-focused information often becomes confusing for people who are just trying to interact with OE as an end user. **We've therefore made the decision to rework the Community Hub into a developer-focused documentation site and to move more user-focused documentation onto a separate platform.**

As a result of this decision, we are planning to remove the high-level "user docs" from this website. We decided to place them on [ZenDesk](https://optimismhelp.zendesk.com/).

We're also using this an opportunity to clean some things up about the existing developer docs. We're starting the process of reviewing everything for outdated information and are coming up with a new, cleaner structure for the docs as a whole. Ideally the new docs will be a lot more information-dense. We'll share our new proposed structure for wider review once we're done with it.

## On translations

Lots of people have volunteered to do translations in various languages (yay!). We haven't managed any large-scale translation projects before so we still have a lot of learning to do. We're planning to test out [Crowdin](https://crowdin.com/) for localization as soon as we're ready to start work on the "new" community hub website. Please keep your eyes peeled for announcements on the [Optimism discord](https://discord.optimism.io) about this if you're interested in contributing to the translation effort.

For existing translation efforts (mainly the CN translation effort in #256), please hold off on making additional translations for now. We will take the work that you've done and adapt it for the new website once Crowdin is available.

## Thank you

Just want to give a huge thank you to everyone who has contributed content to this website! It's amazing to see so many people interested in helping make OE accessible to more people. If you're interested in becoming a more active contributor, feel free to hop in the #contributing channel on the [Optimism discord](https://discord.optimism.io).

Cya around! :heart:

**Comment by qbzzt** [comment]
We have two main choices:

1. [Intercom](https://intercom.help/optimism)
2. [Zen Desk](https://optimismhelp.zendesk.com/hc/en-us)

Which do you think is better? Why?

**Reply by smartcontracts** [reply]
For readers: note that these are the two main choices for where to host user-focused docs.

**Reply by smartcontracts** [reply]
Honestly I'd still love to see some sort of "hold your hand so you can get started with OE" guide.

**Reply by qbzzt** [reply]
You mean like https://optimismhelp.zendesk.com/hc/en-us/sections/4411907003803-Getting-Started ?

**Reply by imtipi** [reply]
maybe intercom,we don't need "Submit a request" and "Comments",just let them go to discord and ask question

**Reply by imtipi** [reply]
oh no,i found a problem. intercom.help are blocked in China...

**Reply by Felix471** [reply]
I discussed this with my UI designer friend and we agree that Intercom one is more straightforward and clear

**Reply by Mark-InCrypto** [reply]
> oh no,i found a problem. intercom.help are blocked in China...

It is okay, cus the vpn is very basic skill in China, for example: mostly exchanges, twitter, even discord.

**Reply by aldifxx** [reply]
In my opinion, the UI design on the intercom is suitable for the user

**Reply by smartcontracts** [reply]
@qbzzt let's try a custom domain and see if still blocked in China.

**Reply by qbzzt** [reply]
> @qbzzt let's try a custom domain and see if still blocked in China.

OK, I asked Ben Wilson to change help.optimism.io to point to Intercom again.

**Reply by qbzzt** [reply]
Done. @imtipi , @Mark-InCrypto , can you check if http://help.optimism.io/ works or not?

**Reply by imtipi** [reply]
> Done. @imtipi , @Mark-InCrypto , can you check if http://help.optimism.io/ works or not?

not working,do you sure you setting custom domain correctly?([Custom Domains for Articles](https://developers.intercom.com/installing-intercom/docs/set-up-your-custom-domain)

**Reply by qbzzt** [reply]
It isn't, I asked our devops to fix that. Sorry, my bad.

**Reply by JackXammie** [reply]
http:// optimism.ai/

**Comment by qbzzt** [comment]
> oh no,i found a problem. intercom.help are blocked in China...

Can you check https://optimismhelp.zendesk.com and https://optimism.helpscoutdocs.com/?

**Reply by imtipi** [reply]
both of them are working
btw,can intercom allow user to use their own domain like userdocs.optimism.io ?
intercom.help blocked but www.intercom.com working
If they don't allow custom domain,we can make a reverse proxy too,but still need to ask for permission

**Reply by Mark-InCrypto** [reply]
Currently the two links work fine from China.
Even needs vpn, it is fine.

**Reply by qbzzt** [reply]
Xie xie

**Comment by comradekingu** [comment]
@smartcontracts Crowdin has no benefits for contributors and is a net detractor from getting contributions. I advice using Weblate.

**Reply by qbzzt** [reply]
What benefits does Weblate have which Crowdin lacks?

**Reply by comradekingu** [reply]
Libre software, can be self-hosted, no spying, Zen-mode to show many languages at once, ability to skip between a large array of strings, active community, etc.

**Reply by qbzzt** [reply]
Thank you. We decided to go with CrowdIn for now. If we decide to switch, the information is all stored in the help center anyway, so we'll be able to do so.

**Reply by comradekingu** [reply]
No, you lose who did what. Moreover, the stringbase is riddled with errors.
Read the terms of service and you find every possible encroachment on freedom there is.
There is a reason individual translators have to be over 18 to even accept dual-licensing their stuff that way.
Edit: Also, don't be worse than Monero.

**Comment by Graciouseats** [comment]
Hello! I see #256 is closed.  Will this effort be reopening now that the new docs are live?  Want to help contribute. Thanks!

**Reply by Mark-InCrypto** [reply]
Hi, the new doc file out these days is more dev files, so it would be better stay in English and currently the users doc as former one is finished for Chinese language, feel free to https://crowdin.com/project/optimism-help-center if you have any more suggestions to the current Chinese language translation.

**Reply by Graciouseats** [reply]
Nice. Thanks! I will mull through them a bit.  What I did read so far looks great.

**Comment by Graciouseats** [comment]
Woah! Celestial signs are the best way to govern..., and practicality.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by Proxtox** [comment]
What languages are most preferred for translation??
